forge-game/src/main/java/forge/game/trigger/TriggerChangesZone.java

```diff
@@ -102,16 +101,13 @@ public final boolean performTest(final Map<AbilityKey, Object> runParams) {
 
         if (hasParam("ValidCard")) {
             Card moved = (Card) runParams.get(AbilityKey.Card);
-            boolean leavesLKIZone = "Battlefield".equals(getParam("Origin"));
-            leavesLKIZone |= "Exile".equals(getParam("Origin")) && (moved.getZone().is(ZoneType.Graveyard) ||
-                    moved.getZone().is(ZoneType.Command) || hasParam("UseLKI"));
 
-            if (leavesLKIZone) {
+            if ("Battlefield".equals(getParam("Origin"))) {
```

Comment:
removed from Exile also should look back aka LKI

Reply:
hmn right, but only partially?
> 603.10a. Some zone-change triggers look back in time. These are leaves-the-battlefield abilities, abilities that trigger when a card leaves a graveyard, and **abilities that trigger when an object that all players can see is put into a hand or library**.

Reply:
But as you said "when a card leaves a graveyard"

You did remove that

Reply:
_Syr Konrad_ currently triggers if I return an Enchantment while _Opalescence_ is in play
I can probably fix that

Reply:
I need to find a smart way though to differentiate if from something like _Breathless Knight_ which also has `Origin$ Graveyard` but it's still ETB trigger